pkg/generate/ack/controller.go

```diff
@@ -169,6 +169,9 @@ var (
 		"CheckNilReferencesPath": func(f *ackmodel.Field, sourceVarName string) string {
 			return code.CheckNilReferencesPath(f, sourceVarName)
 		},
+                "MakeSlice": func (args ...interface{}) []interface{} {
+                        return args
+                },
```

Comment:
@brucegucode can you offer an example of how this type of generic function would be used in a template?

Reply:
@brucegucode OK, I understand the use case better now. How about calling this function `Each`, so that the calling signature would look like this?

```handlebars
{{ range $operationName := Each "RestoreDBInstanceFromDBSnapshot" "CreateDBInstanceReadReplica" }}
```